src/main/java/arona/ListTask.java

```diff
@@ -0,0 +1,25 @@
+package arona;
+
+/**
+ * Allow for user to see the tasklist
+ */
+public class ListTask extends Command {
+    public ListTask(String fullCommand) {
+        super(fullCommand);
+        this.exit = false;
+    }
+
+    /**
+     * Prints the tasklist
+     *
+     * @param taskList The class that contains the tasklist
+     * @param ui UI that handles all the console output
+     * @param storage Handle the loading and saving of data
+     * @throws FileException if file cannot be found
```

Comment:
Perhaps you should remove these thrown exceptions to match your function and avoid confusion?